samples/net/download/sample.yaml

```diff
@@ -7,9 +7,12 @@ tests:
     integration_platforms:
       - nrf9160dk_nrf9160_ns
       - nrf9161dk_nrf9161_ns
-    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
+    platform_allow:
```

Comment:
I think you forgot to remove `.lte` from the test name here.

Reply:
removed the whole test, since we build with tfm as default.